ospf6d/ospf6_asbr.c

```diff
@@ -1113,16 +1147,15 @@ void ospf6_asbr_redistribute_add(int type, ifindex_t ifindex,
 	struct prefix prefix_id;
 	struct route_node *node;
 	char ibuf[16];
-	struct listnode *lnode, *lnnode;
-	struct ospf6_area *oa;
 	struct ospf6_redist *red;
 
 	red = ospf6_redist_lookup(ospf6, type, 0);
 
 	if (!red)
 		return;
 
-	if (!ospf6_zebra_is_redistribute(type, ospf6->vrf_id))
+	if ((type != DEFAULT_ROUTE)
```

Comment:
Extra parenthesis?

Reply:
I think `type != DEFAULT_ROUTE` should have its own parenthesis, which separates it from the `&&` operation in the next line.